docs/beast_setup.rst

```diff
@@ -0,0 +1,73 @@
+Running the BEAST
```

Comment:
Great to have this doc file!  Many thanks for starting it.  I know there is material I will add once I think about it more.